src/commands/deployments/editScmType.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DeploymentsTreeItem } from "vscode-azureappservice";
+import * as appservice from "vscode-azureappservice";
+import { IActionContext } from "vscode-azureextensionui";
+import { SiteTreeItem } from "../../explorer/SiteTreeItem";
+import { ext } from "../../extensionVariables";
+
+export async function editScmType(actionContext: IActionContext, node?: SiteTreeItem | DeploymentsTreeItem): Promise<void> {
+    if (!node) {
+        node = <DeploymentsTreeItem>await ext.tree.showTreeItemPicker([DeploymentsTreeItem.contextValueConnected, DeploymentsTreeItem.contextValueUnconnected]);
+    } else if (node instanceof SiteTreeItem) {
+        actionContext.properties.contextValue = node.contextValue;
```

Comment:
No need to do this. It's handled here: https://github.com/Microsoft/vscode-azuretools/blob/de0a8e1a1c17ff9f5b3b4fecc53b163cb64e3625/ui/src/AzureActionHandler.ts#L30

Reply:
Your knowledge of the codebase is impressive as always 😅